src/layer/marker/Marker.js

```diff
@@ -202,9 +202,13 @@ export var Marker = Layer.extend({
 		var options = this.options,
 		    classToAdd = 'leaflet-zoom-' + (this._zoomAnimated ? 'animated' : 'hide');
 
+		var button = DomUtil.create('button', 'leaflet-marker-button');
 		var icon = options.icon.createIcon(this._icon),
 		    addIcon = false;
 
+		button.setAttribute('aria-haspopup', 'true');

```

Comment:
Why is this needed? Note that [`aria-haspopup="true"`](https://www.w3.org/TR/wai-aria-1.1/#aria-haspopup) "_Indicates the popup is a [`menu`](https://www.w3.org/TR/wai-aria-1.1/#menu)_." (and the leaflet popups are not menus).